### PR TITLE
Structure set Iso Bug fixed

### PR DESCRIPTION
**Pull request by crcrewso**
Referenced in Issue #1597 

Strucutre sets created were not being shifted by the difference between CT center and Isocenter. This fix is tested to work against patient setups without immobilization devices. 

Known issue - if the patient center is offset from the CT center structure set may not match
I consider this improvement sufficient to upstream because currently all patients' structure sets are produced outside the patient ct volume.

### CODE REVIEW

**Comment by SimonBiggs** [comment]
Might you be able to create a small test covering this also? I'll let @pchlap manage this PR though given he's the code owner for this part of the code base 🙂.

**Comment by SimonBiggs** [comment]
You may also need to make sure you have pre-commit installed on your machine as per the overview at:

https://docs.pymedphys.com/contrib/setups/setup-win.html

**Comment by crcrewso** [comment]
> Might you be able to create a small test covering this also? I'll let @pchlap manage this PR though given he's the code owner for this part of the code base slightly_smiling_face.

I'd be happy to, though I'm not sure what that test would look like? Once I get this edge case sorted I'll generate a couple anon versions of the patients that triggered me going down this rabbit hole for the project to use.

**Comment by SimonBiggs** [comment]
> Once I get this edge case sorted I'll generate a couple anon versions of the patients that triggered me going down this rabbit hole for the project to use.

Yup, I think that'd be perfect. Essentially, if the input is an anonned version of the data that flagged it in the first place, then the test just points the relevant functions to that anonned data and makes sure that the relevant component of desired output is as required by the fix.

**Comment by SimonBiggs** [comment]
Here's an example test that @pchlap recently wrote for another PR that is currently open:

https://github.com/pymedphys/pymedphys/blob/057a35de6ca759d612f59f3679ba83fcec5d332f/lib/pymedphys/tests/experimental/pinnacle/test_pinnacle_cli.py#L145-L176

File: lib/pymedphys/_experimental/pinnacle/rtstruct.py [hunk 1]
```diff
@@ -296,7 +299,18 @@ def read_roi(ds, plan):
                     points[1] = round(points[1], 5)
                     points[2] = round(points[2], 5)
 
+                curr_points[0] = curr_points[0] \
+                    - (1*plan.ct_center[0]) \
+                    - (0*plan.iso_center[0])
```
Comment (pchlap) [inline lib/pymedphys/_experimental/pinnacle/rtstruct.py:304]: last part of these equations is multiplying zero * iso_center... Meaning this would always be zero and could be removed for readability?

File: lib/pymedphys/_experimental/pinnacle/rtstruct.py [hunk 2]
```diff
@@ -296,7 +299,18 @@ def read_roi(ds, plan):
                     points[1] = round(points[1], 5)
                     points[2] = round(points[2], 5)
 
+                curr_points[0] = curr_points[0] \
+                    - (1*plan.ct_center[0]) \
```
Comment (pchlap) [inline lib/pymedphys/_experimental/pinnacle/rtstruct.py:303]: A test is failing at this point with "IndexError: list index out of range" indicating that ct_center is empty at least for the test dataset.

Would we always expect there to be a ct_center point?

**Comment by crcrewso** [comment]
CT center and ISO center are user generated. Will find systemic values instead
